backend/pkg/schema/client.go

```diff
@@ -52,7 +52,7 @@ func newClient(cfg config.Schema) (*Client, error) {
 		SetHeader("Accept", "application/vnd.schemaregistry.v1+json").
 		SetHeader("Content-Type", "application/vnd.schemaregistry.v1+json").
 		SetError(&RestError{}).
-		SetTimeout(5 * time.Second)
+		SetTimeout(30 * time.Second)
```

Comment:
Why 30s? Seems like a very long time for a single request to be made. I think not even the frontend would wait for such a long time for the response.

Reply:
We were seeing request timeouts to SR when trying to fetch data from `.../versions` endpoint.
```
... context deadline exceeded (Client.Timeout exceeded while awaiting headers)
``` 
Although could be due to the amount of requests we were sending to SR as well.